pkg/logging/logging_test.go

```diff
@@ -79,4 +80,37 @@ var _ = Describe("logging operations", func() {
 		currentLevel := loggingLevel
 		Expect(currentLevel).To(Equal(GetLoggingLevel()))
 	})
+
```

Comment:
Did you test with missing some options?

For example:

```
		logOptions := &LogOptions{
			MaxAge:     1,
			MaxBackups: 1,
			Compress:   true,
		}
```

In this case, user omit MaxSize so that user expects that multus set default value to MaxSize (not MaxSize=0).

Reply:
Thanks for your point, Done